src/thor/isochrone.cc

```diff
@@ -68,42 +68,62 @@ void Isochrone::Clear() {
   edgestatus_.reset();
 }
 
-// Construct the isotile. Use a grid size based on travel mode.
-// Convert time in minutes to a max distance in meters based on an
-// estimate of max average speed for the travel mode.
+// Construct the isotile. Use a fixed grid size. Convert time in minutes to
+// a max distance in meters based on an estimate of max average speed for
+// the travel mode.
 void Isochrone::ConstructIsoTile(const bool multimodal, const unsigned int max_minutes,
                                  std::vector<baldr::PathLocation>& origin_locations) {
-  float grid_size, max_distance;
+  float max_distance;
   auto max_seconds = max_minutes * 60;
   if (multimodal) {
-    grid_size = 200.0f;
-    max_distance = max_seconds * 70.0f * 0.44704f; // TODO
+    max_distance = max_seconds * 70.0f * kMPHtoMetersPerSec;
   } else if (mode_ == TravelMode::kPedestrian) {
-    grid_size = 200.0f;
-    max_distance = max_seconds * 5.0f * 0.44704f;
+    max_distance = max_seconds * 5.0f * kMPHtoMetersPerSec;
   } else if (mode_ == TravelMode::kBicycle) {
-    grid_size = 200.0f;
-    max_distance = max_seconds * 20.0f * 0.44704f;
+    max_distance = max_seconds * 20.0f * kMPHtoMetersPerSec;
   } else {
     // A driving mode
-    grid_size = 400.0f;
-    max_distance = max_seconds * 70.0f * 0.44704f;
+    max_distance = max_seconds * 70.0f * kMPHtoMetersPerSec;
   }
-  shape_interval_ = grid_size * 0.25f;
 
-  // Form grid for isotiles. Convert grid size to degrees.
-  grid_size /= kMetersPerDegreeLat;
-  float lat = origin_locations[0].latlng_.lat();
+  // Set a fixed grid size (in degrees) - about 220m in latitude
+  float grid_size = 0.002f;
```

Comment:
so previously granularity/precision was fixed and now it will be variable based on the latitude?